### PR TITLE
Dockerfile & Python script

### PR DESCRIPTION
**Pull request by rienafairefr**
I may have been a bit overboard with the contribution there.

The included Dockerfile can generate the disk images as well as page images. Run `docker build -t movinyl .`, then 

- `docker run -v path-with-video-files:/app/PROCESSING_ZONE movinyl disk` will generate the disk images 
- `docker run -v path-with-video-files:/app/PROCESSING_ZONE movinyl page` makes the pages (with auto-generated palette colors, not sure how to fit the GUI color-selection in a Docker workflow)

The permission problem is side-stepped by using gosu and assuming the uid:gid of the host user is 1000:1000 (use `-e RUN_AS=uid:gud` otherwise), this way the files are written inside the mounted volume at /app/PROCESSING_ZONE without being owned by root.

I've regrouped a bunch of scripts into a single script movinyl.py without touching the rest, solving a few bugs along the way (* expansion, treating non-video files etc)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,7 @@ pymdb
 *.png
 *.pyc
 .DS_Store
+.idea/
+pymdb.egg-info/
+infos.txt
+test_files/A Trip to the Moon 1902/images/
```
Comment (Pataclop) [inline .gitignore:13]: Not necessary to add this, and the movie file is included in your commit.

**Review by Pataclop — CHANGES_REQUESTED** [review]
Please remove the movie "A Trip to the Moon 1902.mkv" from the branch, so I can merge. Amazing work, thanks a lot ! Is there a way to process movies in parallel using this method ?

File: .gitignore (hunk 1 above)
Comment (rienafairefr) [reply .gitignore]: OK, I think that's fixed now.

**Comment by rienafairefr** [comment]
Regarding parallel, a bunch of work has already been done with the disk code I think (there's some openmp parallel for). The ffmpeg step (generating the regular snapshots) is simpler in my script, using a single ffmpeg command, and ffmpeg seems to do the multi-thread/cores good enough on its own, so, ... maybe it's easier to do parallel with this, not sure. Maybe adding a multiprocessing/Pool thing on the loop over files `for file in files` in movinyl.py would be enough

**Comment by niniack** [comment]
> Regarding parallel, a bunch of work has already been done with the disk code I think (there's some openmp parallel for). The ffmpeg step (generating the regular snapshots) is simpler in my script, using a single ffmpeg command, and ffmpeg seems to do the multi-thread/cores good enough on its own, so, ... maybe it's easier to do parallel with this, not sure. Maybe adding a multiprocessing/Pool thing on the loop over files `for file in files` in movinyl.py would be enough

PR #12 now uses OpenMP better so that all cores are used throughout generating disks. So, I think its okay if we sequentially do each movie. Otherwise, processing multiple movies simultaneously would just cancel out the parallelization done by OpenMP because they would end up time sharing (is that the right word?) all cores and slowing each other down.

**Pull request merged**